pkg/apis/configuration/v1/types.go

```diff
@@ -579,6 +579,9 @@ type PolicySpec struct {
 	WAF           *WAF           `json:"waf"`
 }
 
+// PolicySpecMap represents a map where policy names map to their specifications.
+type PolicySpecMap map[string]int
```

Comment:
Is it created (and used) solely in the internal package `telemetry`?

Reply:
Yes, but it can removed and turned into map[string]int.